src/main/java/elements/TagImpl.java

```diff
@@ -0,0 +1,25 @@
+package elements;
+
+import org.awaitility.Awaitility;
+import org.openqa.selenium.StaleElementReferenceException;
+import org.openqa.selenium.WebElement;
+import util.ScrollUtils;
+import java.util.concurrent.TimeUnit;
+
+public class TagImpl extends ElementImpl implements Tag {
+
+    public TagImpl(WebElement element) {
+        super(element);
+    }
+
+    @Override
+    public void findTag() {
+        ScrollUtils.scrollUp(getWrappedElement());
+        try {
+            Awaitility.waitAtMost(5, TimeUnit.SECONDS)
+                    .pollInterval(1, TimeUnit.SECONDS)
+                    .until(() -> getWrappedElement().isDisplayed());
+        } catch (StaleElementReferenceException e) {
```

Comment:
What should happen in case of exception?

Reply:
![image](https://user-images.githubusercontent.com/71931457/145395933-9d624bb5-5fc8-41e3-8c7e-c0270c622c38.png)

Reply:
changed to fluent wait